tests/rust_usage_test/tests/integration_test.rs

```diff
@@ -263,6 +263,16 @@ mod lifetime_correctness {
         // this line should compile:
         table.get::<&'static str>(0, None);
     }
+
+    #[test]
+    fn object_lifetime() {
```

Comment:
Please make this a more descriptive test name.

Reply:
Roger, will do.